legal-api/migrations/versions/698885b80fc0_review_update.py

```diff
@@ -0,0 +1,30 @@
+"""review-update
+
+Revision ID: 698885b80fc0
+Revises: 83a110e10979
+Create Date: 2024-10-10 15:17:32.903859
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = '698885b80fc0'
+down_revision = '83a110e10979'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('reviews', sa.Column('contact_details', sa.String(length=150), nullable=True))
```

Comment:
The new field is actually the contact email (aka office email), right? Should it named one of those?

https://github.com/bcgov/sbc-auth/blob/809540257782fad9bc412ac9391b0971acdcd31e/auth-web/src/components/auth/staff/continuation-application/ContinuationApplicationTable.vue#L388

Reply:
We could, I was keeping a generic name if we plan to use email/phone. I will change it to `contact_email` (including in review get call). Can you change UI to use `contactEmail`, then we don't need to do conversion.

Reply:
It will need to be converted _somewhere_ because JS (or at least our linter) doesn't like snake_case. In all other JSON, properties are camelCase and I'd prefer if that was the case here, too.

Reply:
I changed to contactEmail (https://github.com/bcgov/lear/pull/3026/files#diff-2b8fafcba115bab38a344123a02154e6243c84dd946eb49e3d1acd9df922c8ebR160), I dont want to use officeEmail since we have another property called office.

Reply:
Perfect, thanks. I've created ticket 23821 to update the property name in the UI (even though the `reviews` endpoint code is not yet updated).